src/components/tiles/Tile.tsx

```diff
@@ -5,43 +5,22 @@ import { useStyles } from '../../hooks/useStyles';
 import { createStyles } from '../BrandConfigProvider';
 import TileSelectIcon from './icons/tile-select-chevron.svg';
 import { SvgProps } from 'react-native-svg';
-import LinearGradient, {
-  LinearGradientProps,
-} from 'react-native-linear-gradient';
 import { tID } from '../../common/testID';
 
 interface TileProps {
   title: string;
-  chevronGradient?: LinearGradientProps;
   children?: React.ReactNode;
   Icon?: React.FC<SvgProps>;
   id?: string;
   onPress?: () => void;
   testID?: string;
-  tileGradient?: LinearGradientProps;
 }
 
-const defaultTileGradient: LinearGradientProps = {
```

Comment:
@bruce-glazier recently added these. Are we sure we want to remove them?

Reply:
We had gradients where this was ported over from so seemed like something we wanted here too but I get that it doesn't fit the "simple theme" concept we are going for here.

Reply:
Long-term seems like we want to enable either, but I dig this as a short-term change to enable simpleTheme.  One question I have is whether `LinearGradient` is used elsewhere?  AFAIK it's not, so `react-native-linear-gradient` should be removed so that there's one less native iOS pod being packaged with the app.  It also looks like `styles.gradient` could be removed?  Those should ideally be removed in this PR.  

Or, alternatively, I wonder about undoing all the changes in this file and instead making it so `defaultChevronGradient` gets colors of `[primaryColor, primaryColor]`?  This would require moving defaultChevronGradient to be set within the component, because it'd need the theme.

Reply:
Oh, yea, I removed them because the new PMA doc specifically stated that gradients would not be an option.

I'll remove `react-native-linear-gradient` as well.  We can do without the bloat for now and it will be easy to add back whenever we need it.